charts/stable/vikunja/templates/common.yaml

```diff
@@ -1,5 +1,3 @@
-{{- include "tc.common.loader.init" . }}
+{{ include "tc.common.loader.all" . }}
 
 {{- include "vikunja.secrets" . }}
-
-{{ include "tc.common.loader.apply" . }}
```

Comment:
How does changing this, "fixes" the problem of "rediscreds" not found?

I've checked `helm template` for both and rediscreds renders. Also on both it renders 1st (on the printed output atleast)


@Ornias1993

Reply:
`{{- include "tc.common.loader.init" . }}` should responsible of rendering the deps secrets

The only thing that would make sense here is that a `-` is fcking something